SW_Model.c

```diff
@@ -90,7 +95,7 @@ void SW_MDL_construct(void) {
 
 #ifndef STEPWAT
 	/* already set by user-provided seed in steppe */
-	RandSeed(0);
+	RandSeed(0,&markov_rng);

```

Comment:
As suggested in my https://github.com/DrylandEcology/SOILWAT2/issues/226#issuecomment-419931492, the markov weather generator would ideally obtain its own seed. That means a new input for `SOILWAT2`.